code/about.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html id="top">
+
+<head>
+    <meta charset="ut-8" />
+    <meta name="viewport" content="width=device-width">
+    <title>About Us</title>
+    <link rel="stylesheet" href="./style.css" />
+</head>
+
+<header class="simple-header">
+    <nav class="nav">
+        <input type="checkbox" id="checkbox">
+        <label for="checkbox"><a class="menu-button"></a>
```

Comment:
It's not the best idea to wrap `a` tag inside `label`, moreover it's not best idea to add `input` and `label` inside navigation. Let's delete them!

Reply:
If I delete them the "checkbox-hack" to make a drop down menu och click
with only CSS won't work? Should I remove this function then? I know it is a bit of a "hack", but I liked making the page working well on mobile with only css

Reply:
Ahh, true, I didn't quite catch that one. In that case let's leave it as it is ;)